rainbow-me/browser-extension
# feat: adds initial loading for tokens and activity

Fixes BX-296
Figma link (if any): https://www.figma.com/file/hzfwKOddfRmeVO4KUsG9wM/extensions-v2?node-id=5851%3A205703&t=irHqKe3lChzPHvPZ-4

## What changed (plus any additional context for devs)
- Adds initialLoading states for tokens and activity
- Balance and tokens/activity prices load at different rates, I decided to show each whenever they are available rather than have them all load at the same time just on the principle of "if we have the data, we should show it and not artificially hide".


## Screen recordings / screenshots
<img width="359" alt="image" src="https://user-images.githubusercontent.com/3009331/219711325-474cb8b3-bc86-4bda-b39a-443f48252709.png">



## What to test
- Load a wallet where you need to do an "initial loading" of tokens or activities
  -  see loading shimmers
- You should not see loading shimmers during any re/fetching. Only on initial load.


## Final checklist
